github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-postsubmits.yaml

```diff
@@ -195,3 +195,45 @@ postsubmits:
         resources:
           requests:
             memory: "8Gi"
+  - name: push-cdi-goveralls
+    cluster: kubevirt-prow-control-plane
+    annotations:
+      testgrid-create-test-group: "false"
+    always_run: true
+    optional: true
```

Comment:
I'm not sure if `optional: true` makes sense on a postsubmit. You can drop this.

Reply:
Makes sense, done :+1: